script/Ciel/Day 12/12_01_CIEL12_3_1.txt

```diff
@@ -271,42 +286,48 @@ But I'm glad that I came.
 [sha:f68d218b0e99ff5e2bcdaf06720834df2004558e]{
 -- Page 169, Offset 31317.
 -- 先輩も楽しみにしてくれていたんだ。
-It seems even Senpai was looking forward to it.
+It seems like Senpai was looking forward to this too.
 }
 [sha:7305de1b16f02e59f2b89c58c67f7746abfb3d1c]{
 -- Page 169, Offset 31318.
 -- なら、こんな眩暈ぐらい我慢して、三人で楽しく、ありふれた日々のように―――――
-In that case, I'll put up with this vertigo, and the three of us can enjoy this mundane day together―――――
+I've got to power through this vertigo so that the three of us can have fun together,
+ and enjoy this ordinary day to the fullest―――
 }
 [sha:dcce50ab1c619e397bf65c98324f4a28834c578b]{
 -- Page 170, Offset 31319.
 -- 気がつくと、ベッドで眠っている自分がいた。
-When I come to, I find myself laid down on a bed.
+// 00_00 callback?
+The next thing I know, I'm lying in bed.
 }
 [sha:6f4f9ea5e3dad3ed4c9319a3274e6ec3900202e6]{
 -- Page 171, Offset 31320.
 -- 「あ―――れ」
-"―――Huh?"
+// Pretty long VA space here
+"Huh... what―――"
```

Comment:
Maybe like

>What... the―――

is more appropriate.

Reply:
Ehh, that makes it sound like there's more to it to me.

Reply:
Well, it's enough to convey surprise. The current one seems so haphazard.
Maybe something like this
>Hh―――uhh?

or this

>Wh―――what?

Reply:
Then I think "what―――the [hell?]" is still superior, since it does what yours does much more efficiently, while preserving some continuity between them like how it is in JP.